lib/elixir/test/elixir/inspect/algebra_test.exs

```diff
@@ -258,19 +245,16 @@ defmodule Inspect.AlgebraTest do
     assert force_unfit("ab") == {:doc_force, "ab"}
     assert force_unfit(empty()) == {:doc_force, empty()}
 
-    # Consistent with definitions
-    assert next_break_fits("ab") == {:doc_fits, "ab", :enabled}
-    assert next_break_fits(empty()) == {:doc_fits, empty(), :enabled}
-    assert next_break_fits("ab", :disabled) == {:doc_fits, "ab", :disabled}
-    assert next_break_fits(empty(), :disabled) == {:doc_fits, empty(), :disabled}
-
     # Consistent formatting
-    doc = force_unfit(concat(glue(glue(glue("hello", "a"), "b"), "c"), "d"))
-    assert render(doc, 20) == "hello\na\nb\ncd"
-    assert render(next_break_fits(doc, :enabled), 20) == "hello a b cd"
+    doc = force_unfit(glue(glue("hello", "a"), "b"))
+    assert render(doc, 20) == "hello\na\nb"
+    assert render(doc |> glue("c") |> group(), 20) == "hello\na\nb\nc"
+
+    assert render(doc |> group(:optimistic) |> glue("c") |> group(), 20) ==
+             "hello\na\nb c"
 
-    assert render(next_break_fits(next_break_fits(doc, :enabled), :disabled), 20) ==
-             "hello\na\nb\ncd"
+    assert render(doc |> group(:optimistic) |> glue("c") |> group(:pessimistic), 20) ==
+             "hello\na\nb c"
```

Comment:
I don't think we're really testing `:pessimistic` here, since this the same as the test above, right?